app/_hub/datadome/kong-plugin-datadome/_index.md

```diff
@@ -3,4 +3,62 @@ The DataDome plugin is developed in Lua and integrates smoothly with Kong.
 This plugin will analyse in real time any API requests received. For each request, an api call is sent to DataDome Bot & Fraud Protection Platform to validate if this is legitimate or not.
 The only thing to configure is your DataDome Server Side Key.
 
-[This integration guide](https://docs.datadome.co/docs/kong-plugin) describes how to integrate DataDome with Kong API Gateway.
\ No newline at end of file
+## How it works
+
+The DataDome plugin will hook into the request/response of an API request and it is executed for every request from a client and before it is being proxied to the upstream on the access phase (see details [here](https://openresty-reference.readthedocs.io/en/latest/Directives/)).
+
+## How to install
+
+Custom plugins can be installed via LuaRocks. A Lua plugin is distributed in `.rock` format, which is
+a self-contained package that can be installed locally or from a remote server.
+
+If you used one of the official {{site.base_gateway}} installation packages, the LuaRocks utility
+should already be installed in your system.
+Install the `.rock` in your LuaRocks tree, that is, the directory in which LuaRocks
+installs Lua modules.
+
+1. Install the DataDome plugin:
+
+    ```sh
+    luarocks install kong-plugin-datadome
+    ```
+
+2. Update your loaded plugins list in {{site.base_gateway}}.
+
+    In your `kong.conf`, append `datadome` to the `plugins` field. Make sure the field is not commented out.
+
+    ```yaml
+    plugins = bundled,datadome              # Comma-separated list of plugins this node
+                                            # should load. By default, only plugins
+                                            # bundled in official distributions are
+                                            # loaded via the `bundled` keyword.
+    ```
+
+3. Restart {{site.base_gateway}}:
+
+    ```sh
+    kong restart
+    ```
+
+## Using the plugin
+
+First of all, make sure you have your DataDome Server Side Key. It is available inside your [Dashboard > Integrations](https://app.datadome.co/dashboard/management/integrations).
+
+#### Kong Gateway (OSS)
+If you already have an API configured, execute the command below replacing the following `<values>`:
```

Comment:
- Use correct heading level
- Prefer active form instead of passive
- Use explicit terms instead of `<values>` as a pattern

```suggestion
### Kong Gateway (OSS)
If you already configured an API, execute the command below after replacing `<YOUR_API>` with the name of your API and `<server_side_key>` with your DataDome server-side key.
```

Reply:
On https://docs.datadome.co/docs/kong-plugin, it's called Kong Open Source.
Which one is the official name?

Reply:
they use in both ways (https://docs.konghq.com/gateway/latest/#on-premises), but I would update the public doc like here Kong Gateway (OSS)  or Kong Gateway OSS (Open Source) if we want to be really explicit